tests/ci/cdk/cdk/bm_framework_stack.py

```diff
@@ -5,25 +5,35 @@
 import boto3
 
 from botocore.exceptions import ClientError
-from aws_cdk import Duration, Stack, aws_ec2 as ec2, aws_codebuild as codebuild, aws_iam as iam, aws_logs as logs
+from aws_cdk import (
+    Duration,
+    Stack,
+    aws_ec2 as ec2,
+    aws_codebuild as codebuild,
+    aws_iam as iam,
+    aws_logs as logs,
+)
 from constructs import Construct
 
 from cdk.components import PruneStaleGitHubBuilds
-from util.metadata import AWS_ACCOUNT, AWS_REGION, GITHUB_REPO_OWNER, GITHUB_REPO_NAME
-from util.iam_policies import code_build_batch_policy_in_json, ec2_bm_framework_policies_in_json, \
-                                ssm_bm_framework_policies_in_json, ecr_power_user_policy_in_json
+from util.metadata import GITHUB_REPO_OWNER, GITHUB_REPO_NAME
+from util.iam_policies import (
+    code_build_batch_policy_in_json,
+    ec2_bm_framework_policies_in_json,
+    ssm_bm_framework_policies_in_json,
+    ecr_power_user_policy_in_json,
+)
 from util.build_spec_loader import BuildSpecLoader
 
 # detailed documentation can be found here: https://docs.aws.amazon.com/cdk/api/latest/docs/aws-ec2-readme.html
 
+
 class BmFrameworkStack(Stack):
     """Define a stack used to create a CodeBuild instance on which to execute the AWS-LC benchmarking framework"""
```

Comment:
Why doesn't this stack have logic for staging account versus non-staging account?